app/models/admin-sales-discounted.js

```diff
@@ -2,8 +2,9 @@ import attr from 'ember-data/attr';
 import ModelBase from 'open-event-frontend/models/base';
 
 export default ModelBase.extend({
-  code      : attr('string'),
-  email     : attr('string'),
-  eventName : attr('string'),
-  sales     : attr()
+  code            : attr('string'),
+  email           : attr('string'),
+  eventName       : attr('string'),
+  paymentCurrency : attr('string'),
```

Comment:
is this returned by the server? Have you verified?

Reply:
@CosmicCoder no. I've made changes on my local server. I'll push it once you confirm this is relevant issue.

Reply:
Opened a PR on server